<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchFlat</class>
 <widget class="QDialog" name="SearchFlat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>111</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>211</width>
    <height>111</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>211</width>
    <height>111</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Поиск в картотеке</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/res.qrc">
    <normaloff>:/new/window_icons/icons/card.png</normaloff>:/new/window_icons/icons/card.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background: rgb(225, 225, 225);
}

QPushButton{
background: rgb(210,210,210);
border: none;
border-radius: 4px;
font-family: Calibri;
font-size: 10pt;
}
QPushButton:hover{
background: rgb(195, 195, 195);
}

QLineEdit {
border: none;
}

QLabel{
font-size: 11pt;
font-family: Calibri;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество комнат:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Этаж:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rooms_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="block_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource/res.qrc"/>
 </resources>
 <connections/>
</ui>
